src/client/compiler/parser/Parser.ts

```diff
@@ -1037,6 +1039,7 @@ export class Parser {
     }
 
     // -, not, this, super, a.b.c[][].d, a.b(), b() (== this.b()), super.b(), super()
+    // FIXME: @RALF add tilde support

```

Comment:
That's one of the reasons this MR is still WIP.

Reply:
See Parser.parseUnary()